content/influxdb/v2.2/api-guide/tutorials/client-library-starter/python.md

```diff
@@ -0,0 +1,720 @@
+---
+title: Python client library starter
+seotitle: Use Python client library to build a sample application
+list_title: Python
+description: >
+  Build an application that writes, queries, and manages devices with the InfluxDB
+  client library for Python.
+weight: 3
+menu:
+  influxdb_2_2:
+    identifier: client-library-starter-py
+    name: Python
+    parent: Client library starter
+influxdb/cloud/tags: [api]
+---
+
+{{% api/iot-starter-intro %}}
+- How to use the InfluxData UI libraries to format data and create visualizations.
```

Comment:
Does this render correctly? The shortcode ends with an unordered list, so I assume this would render a separate unordered list after the shortcode. If it looks ok, then it's probably fine, but I'm thinking there will be a gap before this additional list item.